change/@fluentui-react-combobox-057ab96a-491a-422f-8b6b-ee92954175ce.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30033&commitId=ed1dd4852648d9b97e7b86c10cee4ff88b81fea7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size with async content.floating element is within the boundary.chromium.png|3434| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>